### PR TITLE
chore: move scss grunt tasks to npm scripts

### PR DESCRIPTION
**Pull request by BrandonOCasey**
Move all the scss login from grunt into npm scripts, so that one day we can remove grunt

### CODE REVIEW

File: build/grunt.js [hunk 1]
```diff
@@ -85,19 +84,6 @@ module.exports = function(grunt) {
     },
     dist: {},
     watch: {
-      dist: {
-        files: [
-          'build/temp/video.js',
-          'build/temp/alt/video.novtt.js',
-          'build/temp/video-js.css',
```
Comment (BrandonOCasey) [inline build/grunt.js]: We no longer need this since `rollup` handles the the js files and `node-sass` will handle the css

File: build/grunt.js [hunk 2]
```diff
@@ -128,27 +114,7 @@ module.exports = function(grunt) {
       a11y:  { src: 'sandbox/descriptions.html.example', dest: 'sandbox/descriptions.test-a11y.html' }, // Can only test a file with a .html or .htm extension
       examples: { cwd: 'docs/examples/', src: ['**/**'], dest: 'dist/examples/', expand: true, filter: 'isFile' }
     },
-    cssmin: {
```
Comment (BrandonOCasey) [inline build/grunt.js]: add as a npm script under `cssmin`

File: build/grunt.js [hunk 3]
```diff
@@ -200,14 +166,6 @@ module.exports = function(grunt) {
         dest: 'dist/video-js-' + version.full + '.zip'
       }
     },
-    version: {
```
Comment (BrandonOCasey) [inline build/grunt.js]: i'm not sure that this actually did anything, I see no difference in master with it removed

File: package.json [hunk 4]
```diff
@@ -20,9 +20,21 @@
     "changelog": "conventional-changelog -p videojs -i CHANGELOG.md -s",
     "build": "grunt dist",
     "minify": "babel-node build/minify.js",
+    "cssmin": "npm-run-all cssmin:*",
+    "cssmin:cdn": "cleancss dist/alt/video-js-cdn.css -o dist/alt/video-js-cdn.min.css",
+    "cssmin:default": "cleancss dist/video-js.css -o dist/video-js.min.css",
+    "sass": "npm-run-all sass:* autoprefixer",
+    "sass:cdn": "node-sass src/css/vjs-cdn.scss dist/alt/video-js-cdn.css",
+    "sass:default": "node-sass src/css/vjs.scss dist/video-js.css",
```
Comment (gkatsev) [inline package.json]: Tests still rely on `build/temp/video-js.css`, we should update that and also make sure that there aren't any other uses of `build/temp/video-js.css` like in sandbox or doc files.

Comment (BrandonOCasey) [reply package.json]: good call

File: build/grunt.js [hunk 5]
```diff
@@ -113,41 +99,11 @@ module.exports = function(grunt) {
       }
     },
     copy: {
-      minor: {
-        files: [
-          {expand: true, cwd: 'build/temp/', src: ['*'], dest: 'dist/'+version.majorMinor+'/', filter: 'isFile'} // includes files in path
-        ]
```
Comment (BrandonOCasey) [inline build/grunt.js]: We don't use these

File: build/grunt.js [hunk 6]
```diff
@@ -200,50 +156,7 @@ module.exports = function(grunt) {
...
-    },
     browserify: {
-      build: {
-        options: browserifyGruntOptions(),
-        files: {
-          'build/temp/video.js': ['es5/video.js']
-        }
-      },
-      buildnovtt: {
-        options: browserifyGruntOptions({transform: [
```
Comment (BrandonOCasey) [inline build/grunt.js]: We don't use any of these anymore either

File: build/grunt.js [hunk 7]
```diff
@@ -276,11 +189,6 @@ module.exports = function(grunt) {
         src: 'test/coverage/lcov.info'
       }
     },
-    concat: {
-      options: {
```
Comment (BrandonOCasey) [inline build/grunt.js]: not used

File: test/karma.conf.js [hunk 8]
```diff
@@ -29,7 +29,6 @@ module.exports = function(config) {
 
     // Using precompiled tests
     // files: [
-    //   '../build/temp/video-js.css',
```
Comment (BrandonOCasey) [inline test/karma.conf.js]: Should we just remove this comment

File: package.json [hunk 9]
```diff
@@ -92,13 +102,10 @@
     "grunt-contrib-concat": "^1.0.1",
```
Comment (gkatsev) [inline package.json]: I think this isn't used anymore and could be removed

File: test/karma.conf.js (hunk 8 above)
Comment (gkatsev) [reply test/karma.conf.js]: We may as well.

File: build/grunt.js [hunk 10]
```diff
@@ -312,25 +218,19 @@ module.exports = function(grunt) {
         'browserify:watch'
       ]
     },
-    usebanner: {
```
Comment (gkatsev) [inline build/grunt.js]: I guess we can remove `grunt-banner` as well, though, this made me realize we still keep the vttjs license in the novtt file banner (though we shouldn't bother with it in this PR)

Comment (gkatsev) [reply build/grunt.js]: https://github.com/videojs/video.js/issues/5526

File: sandbox/vertical-volume.html.example [hunk 11]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```
Comment (gkatsev) [inline sandbox/vertical-volume.html.example]: seems useful, though I tend to just manually call `player.controlBar.addChild('volumePanel', {inline: false});`

**Pull request merged**